inc/command/class-command.php

```diff
@@ -188,14 +203,224 @@ protected function phpunit( InputInterface $input, OutputInterface $output ) {
 		return $this->run_command( $input, $output, 'vendor/bin/phpunit', $options );
 	}
 
+	/**
+	 * Runs Codecept with zero config by default.
+	 *
+	 * @param InputInterface $input
+	 * @param OutputInterface $output
+	 * @return int
+	 */
+	protected function codecept( InputInterface $input, OutputInterface $output ) {
+		$options = $input->getArgument( 'options' );
+		$module = $input->getOption( 'module' );
+		$run_headless_browser = $input->getOption( 'browser' );
+		$use_chassis = $input->getOption( 'chassis' );
+		$tests_folder = $module !== 'project' ? "altis/$module/tests" : '../tests';
+		$project_subdomain = $this->get_project_subdomain();
+
+		// Write the default config.
+		$config = [
+			'paths' => [
+				'tests' => $tests_folder,
+				'output' => 'altis/dev-tools/tests/_output',
+				'data' => 'altis/dev-tools/tests/_data',
+				'support' => 'altis/dev-tools/tests/_support',
```

Comment:
```suggestion
				'support' => '../tests/_support',
```

We can probably get away with the data path but the output and support should be under the tests directory path. I know that they have the `.gitignore` in them but for a package installed from packagist and not as a git checkout the git ignore rules will have no effect when it comes to composer determining if a package has been modified.

Reply:
I agree about the output, all output should be added somewhere visible and outside of the vendor dir, however, support has generated AND updated files, so would keep it in dev-tools so all modules can make use of it. It is not auto generated unless `build` command is invoked which isn't something we'd be doing regularly, unlike `output`.

Also it seems like it auto-creates the directory, so we don't need to care about that.

Reply:
Are the generated files not specific to the tests they belong to? Would we end up in a position where to add tests to one module we would have to update this one?

Does build just have to be run the once when you add a new suite or is it required after creating new tests too? Sorry for all the questions, just trying to understand the implications of this.

My issue is that this config will be used by projects to test too, and we really don't want someone's tests to cause changes in the dev-tools package directory under vendor.

Reply:
- Generated files are not specific to specific tests, no, they're helpers that you can use with any suite. Sort of imports that you can reference from anywhere.
- You don't need to build or generate anything to create a new tests. Just generating the test itself is enough.
- Projects can use their own helpers if need be, and I assume they'll be able to extend ours and add their own methods where needed. FWIW all the methods are added to traits which any helper can use.